daml-foundations/daml-ghc/src/DA/Daml/GHC/Compiler/Options.hs

```diff
@@ -147,6 +148,7 @@ defaultOptions mbVersion =
         , optPackageDbs = []
         , optMbPackageName = Nothing
         , optWriteInterface = False
+        , optIfaceDir = "."
```

Comment:
Why should we default to putting them into the current directory instead of defaulting to .interfaces?

Reply:
👍

Reply:
I removed the option. It's now fixed ".interfaces".